platform-ajax/src/main/typescript/com/softicar/platform/ajax/engine/request/AjaxRequestMessage.ts

```diff
@@ -128,6 +144,22 @@ class AjaxRequestMessage {
 		}
 	}
 
+	private isSameDeltaDirections(other: AjaxRequestMessage) {
```

Comment:
i think the type definitions are useless, and isn't there a more explicit way to convert to number than to use the unary `+` operator?

Reply:
there is. we can just use `Number(someString)` instead.